src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,25 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    public static final User[] users = new User[] {
```

Comment:
Make it private field, cause we dont want to expose our users everywhere
```suggestion
    private static final User[] users = new User[] {
```